### DISCUSSION TITLE
could you add font dirtectory in papermod

### DISCUSSION THREAD

**Discussion by hiDaDeng** [adityatelange/hugo-PaperMod#1185] [Q&A]
I want change the font of papermod,but i dont know how to do. Could you add font dirtectory ,so we can overwrite the ttf file to change the font

**Comment by wildserver** [comment]
https://forum.wildserver.ru/viewtopic.php?t=18

**Comment by williameast** [comment]
I did this and it worked fine for me:

first find the font you want, then check it exists in [Google Fonts](https://fonts.google.com/).

this will produce the HTML and CSS code you need, when you select the font (press add xxx on the right side of each font you want)

add the HTML portion to `themes/PaperMod/layouts/partials/extend_head.html`  after the commented section, and then the CSS portion to `hemes/papermod/assets/extended/blank.css`. 
 
then it should change directly.

**Comment by lokesh-krishna** [comment]
This did work but it seems like not all weights are brought into play. How can I make sure that the bold bits actually use the bold weight of the font and not just make the regular weight appear bold?

**Comment by be4zad** [comment]
First add your fonts to `/static/fonts/` then create `/assets/css/extended/blank.css` and paste your font-face or other css things, but set url to `/fonts`
